toolkit/offchain/tests/integration_tests.rs

```diff
@@ -56,7 +56,8 @@ const EVE_ADDRESS: &str = "addr_test1vzzt5pwz3pum9xdgxalxyy52m3aqur0n43pcl727l37
 const REWARDS_TOKEN_POLICY_ID: PolicyId =
 	PolicyId(hex!("1fab25f376bc49a181d03a869ee8eaa3157a3a3d242a619ca7995b2b"));
 
-const REWARDS_TOKEN_ASSET_NAME_STR: &str = "5043526f636b73";
+// Reward token
+const REWARDS_TOKEN_ASSET_NAME_STR: &str = "52657761726420746f6b656e";
```

Comment:
I've changed this token name (and initial transaction), so it matches local-environment. This means it is easy to have same utxo (tx hashes) when running scenarios in local-environment and in tests here.